examples/5_DataManagement/build_an_image_dataset.py

```diff
@@ -0,0 +1,211 @@
+""" Build an Image Dataset in TensorFlow.
+
+For this example, you need to make your own set of images (JPEG).
+We will show 2 different ways to build that dataset:
+
+- From a root folder, that will have a sub-folder containing images for each class
+    ```
+    ROOT_FOLDER
+       |-------- SUBFOLDER (CLASS 0)
+       |             |
+       |             | ----- image1.jpg
+       |             | ----- image2.jpg
+       |             | ----- etc...
+       |             
+       |-------- SUBFOLDER (CLASS 1)
+       |             |
+       |             | ----- image1.jpg
+       |             | ----- image2.jpg
+       |             | ----- etc...
+    ```
+
+- From a plain text file, that will list all images with their class ID:
+    ```
+    /path/to/image/1.jpg CLASS_ID
+    /path/to/image/2.jpg CLASS_ID
+    /path/to/image/3.jpg CLASS_ID
+    /path/to/image/4.jpg CLASS_ID
+    etc...
+    ```
+
+Below, there are some parameters that you need to change (Marked 'CHANGE HERE'), 
+such as the dataset path.
+
+Author: Aymeric Damien
+Project: https://github.com/aymericdamien/TensorFlow-Examples/
+"""
+from __future__ import print_function
+
+import tensorflow as tf
+import os
+
+# Dataset Parameters - CHANGE HERE
+MODE = 'folder' # or 'file', if you choose a plain text file (see above).
+DATASET_PATH = '/path/to/dataset/' # the dataset file or root folder path.
+
+# Image Parameters
+N_CLASSES = 2 # CHANGE HERE, total number of classes
+IMG_HEIGHT = 64 # CHANGE HERE, the image height to be resized to
+IMG_WIDTH = 64 # CHANGE HERE, the image width to be resized to
+CHANNELS = 3 # The 3 color channels, change to 1 if grayscale
+
+
+# Reading the dataset
+# 2 modes: 'file' or 'folder'
+def read_images(dataset_path, mode, batch_size):
+    imagepaths, labels = list(), list()
+    if mode == 'file':
+        # Read dataset file
+        data = open(dataset_path, 'r').read().splitlines()
```

Comment:
how to fix this?


In [46]: def read_images(dataset_path, mode, batch_size):
    ...:     imagepaths, labels = list(), list()
    ...:     if mode == 'fyp.txt':
    ...:     data=open(dataset_path, 'r').read().splitlines()
  File "<ipython-input-46-bffd7f50e56c>", line 4
    data=open(dataset_path, 'r').read().splitlines()
       ^
IndentationError: expected an indented block